src/layout-pack.js

```diff
@@ -82,6 +105,12 @@ var PackLayout = React.createClass({
     },
 
     componentDidUpdate: function() {
+        // When image finish loading run reposition
+        this.image = this.el.querySelectorAll('img');
+        for (var i = this.image.length - 1; i >= 0; i--) {
+            this.image[i].onload = this.reposition;
```

Comment:
This will override any existing user-specified onload handlers... Could we instead use `removeEventListener` + `addEventListener`?